File: examples/Python-parte1/simpleudpserver.py
import socket

i = 0
HOST = ''
PORT = 6789
sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
sock.bind((HOST, PORT))
while True:
    i += 1
    print("Esperando Msg {s} ...".format(s=i))
    data, address = sock.recvfrom(1024)
    h, p = address
    print("Cliente: {s} - Porta: {t}\nMsg: {u}".format(s=h, t=p, u=data.decode('utf-8')))

File: examples/NodeJS-parte1/simpleudpServer.js
var PORT = 7895;
var HOST = '127.0.0.1';
var i = 1

var dgram = require('dgram');
var server = dgram.createSocket('udp4');

server.on('listening', function() {
  console.info(`Esperando Msg ${i} ...`);
});

server.on('message', function(message, remote) {
 
 var msg = message.toString();
 msg = msg.toUpperCase();
 server.send(msg, 0, message.length, remote.port, remote.address);
 console.info(`Cliente: ${remote.address} - Porta: ${remote.port}`);
 console.info(`Msg: ${message}`);
 i+=1
 console.info(`Esperando Msg ${i} ...`);
});

server.bind(PORT, HOST);
